packages/page-accounts/src/Accounts/index.spec.ts

```diff
@@ -50,6 +51,13 @@ describe('Accounts page', () => {
 
       expect(noAccountsMessage).not.toBeNull();
     });
+
+    it('no summary displays', () => {
+      accountsPage.renderPage([]);
```

Comment:
This line can be moved do a `beforeEach` in `when no accounts` describe/context block.

Reply:
Indeed

Reply:
Done, thanks 👍